docs/reference/rollup/understanding-groups.asciidoc

```diff
@@ -7,7 +7,7 @@ experimental[]
 
 To preserve flexibility, Rollup Jobs are defined based on how future queries may need to use the data.  Traditionally, systems force
 the admin to make decisions about what metrics to rollup and on what interval.  E.g. The average of `cpu_time` on an hourly basis.  This
-is limiting; if, at a future date, the admin wishes to see the average of `cpu_time` on an hourly basis _and partitioned by `host_name`_,
+is limiting, in the future if the admin wishes to see the average of `cpu_time` on an hourly basis and partitioned by `host_name`,
```

Comment:
Thanks for the clarification. I'm going to incorporate a suggestion that fixes the formatting without removing the semicolon.

```suggestion
is limiting; if, in the future, the admin wishes to see the average of `cpu_time` on an hourly basis _and_ partitioned by `host_name`,
```

Reply:
Hi @jrodewig thanks for the suggestion. I have updated the change.